cmd/image-builder-db-test/main_test.go

```diff
@@ -110,9 +110,23 @@ func testMigration(t *testing.T) {
 
 	migrateOneStep(t) // migrate to step 3
 
+	migrateOneStep(t) // migrate to step 4
+
+	imageName := "MyImageName"
+
+	// inserting image_name should succeed
+	insert = "INSERT INTO composes(job_id, request, created_at, account_number, org_id, image_name) VALUES ($1, $2, CURRENT_TIMESTAMP, $3, $4, $5)"
+	_, err = conn.Exec(context.Background(), insert, uuid.New().String(), "{}", ANR3, ORGID1, &imageName)
+	require.NoError(t, err)
+
+	// inserting empty image_name should succeed
+	insert = "INSERT INTO composes(job_id, request, created_at, account_number, org_id, image_name) VALUES ($1, $2, CURRENT_TIMESTAMP, $3, $4, $5)"
+	_, err = conn.Exec(context.Background(), insert, uuid.New().String(), "{}", ANR3, ORGID1, nil)
+	require.NoError(t, err)
+
 	// Verify that after migration step 3 adding a compose request to the db requires a non empty account number.
 	d, err := db.InitDBConnectionPool(connStr(t))
-	err = d.InsertCompose(uuid.New().String(), "", ORGID1, []byte("{}"))
+	err = d.InsertCompose(uuid.New().String(), "", ORGID1, &imageName, []byte("{}"))
 	require.Error(t, err)
```

Comment:
Ah I only now noticed this, these 4 lines should move up to in between migration 3 and 4 :/

Reply:
Moved